torch/_refs/__init__.py

```diff
@@ -1432,6 +1432,20 @@ def le(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
     return prims.le(a, b)
 
 
+@_make_elementwise_binary_reference(
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+    supports_lhs_python_scalar=False,
+    supports_rhs_python_scalar=False,
+)
+def logaddexp(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
+    # Nb. this implementation does nto distribute the gradients evenly when a == b
+    mask = a >= b
+    max_ = torch.where(mask, a, b)
```

Comment:
I'm not sure 2 masking ops are better than one `abs`, but I don't think it matters much.

Reply:
it's 1 masking op vs 1 abs and a neg. It should be marginally better, but yeah

Reply:
Also, note that nvcc and may be able to fuse the two wheres into just one flow control op.